### PR TITLE
Feat/iu 20 gruped search results unmerged commits

### PR DESCRIPTION
**Pull request by mikowhy**
new PR for https://github.com/unicef/iogt/issues/83
---

https://github.com/unicef/iogt/pull/222 was marged with core logic of grouping search results while there were additional unmerged commits in https://github.com/unicef/iogt/pull/122

+ minor changes after code review

### CODE REVIEW

File: search/views.py [hunk 1]
```diff
@@ -4,21 +4,23 @@
 from iogt.settings.base import SEARCH_RESULTS_PER_PAGE
 from wagtail.search.models import Query
 
+from questionnaires.models import Survey, Quiz
+
 
 def search(request):
-    search_groups = (Article, Section)  # add models which should be searchable
+    search_groups = (Article, Section, Survey, Quiz)
```
Comment (ehmadzubair) [inline search/views.py:11]: Do we also need to add Poll?

File: search/views.py [hunk 2]
```diff
@@ -4,21 +4,23 @@
...
     search_query = request.GET.get("query")
-    page = request.GET.get("page", 1)
+    page = int(request.GET.get("page", 1))
 
     results = {"search_query": search_query, "search_groups": {}}
 
     # Search
     for search_group in search_groups:
-        search_results_no = 0
+        search_results_amount = 0
```
Comment (ehmadzubair) [inline search/views.py:20]: I think amount suggests a monetary notion to the variable. Can we revert it or make it search_results_count?

File: search/views.py (hunk 1 above)
Comment (dannyparsons) [reply search/views.py:11]: Yes please add `Poll`

File: iogt/urls.py [hunk 3]
```diff
@@ -28,7 +28,6 @@
     path("external-link/", TransitionPageView.as_view(), name="external-link"),
     path('messaging/', include('messaging.urls'), name='messaging-urls'),
     path('wagtail-transfer/', include(wagtailtransfer_urls)),
-    path("external-link/", TransitionPageView.as_view(), name="external-link"),
```
Comment (dannyparsons) [inline iogt/urls.py:31]: Should this have been removed?

Comment (mikowhy) [reply iogt/urls.py:31]: it was a repeating from line 28

File: search/views.py (hunk 1 above)
Comment (mikowhy) [reply search/views.py:11]: ok, will be added

File: search/views.py (hunk 2 above)
Comment (mikowhy) [reply search/views.py:20]: ok, will be changed

Comment (mikowhy) [reply search/views.py]: ✅

File: search/views.py (hunk 1 above)
Comment (mikowhy) [reply search/views.py]: ✅

**Comment by mikowhy** [comment]
@ehmadzubair could you please check? thx

**Pull request merged**